### DISCUSSION TITLE
What's the segmentation type?

### DISCUSSION THREAD

**Discussion by deathmukh** [neuroneural/brainchop#26] [Q&A]
So as far as i know, image segmentation is done using four techniques: 
1. Region-based segmentation
2. Edge detection segmentation
3. Thresholding 
4. Clustering 
What technique does brainchop use?

**Comment by Mmasoud1** [comment]
Hi @deathmukh,  we are using deep learning (DL) in which we train a model on segmentation task and then use the trained model for new data segmentation.
